wavelength-plugins/splatoon.js

```diff
@@ -0,0 +1,192 @@
+/****************************
+ * Splatoon Plug-in for PS	*
+ * Created by Insist		*
+ ****************************/
+
+"use strict";
+
+let SPLATFEST = false;
+let splatfestTeams = [];
```

Comment:
Id combine this into one object such as
```javascript
let SPLATFEST = {alpha: null, bravo: null, active: false};
```
When setting up a splatfest, require that the two teams are passed to the command creating it.